docs/source/gbd2017_models/causes/covid_19/index.rst

```diff
@@ -11,35 +11,285 @@ Covid-19 Custom SEIR Vivarium Cause Model
+
+In the same time-step for which a simulant is initialized into the :math:`E`, :math:`I_1`, or :math:`I_2` covid-19 model states or transitions into the :math:`E` model state, the age-specific IFR should be used to determine if that simulant will die of covid-19, such that the IFR shown in the table below represents the probability that an infected individual dies from covid-19. 
+
+.. csv-table:: Infection Fatality Ratios
+	:file: ifr_data.csv
+	:header-rows: 1
+
+.. note::
+
+	This data was obtained from an email communication with Abie from May 5, 2020. These values may need to be updated as more recent numbers become available.  
+
 Transition Data
 ^^^^^^^^^^^^^^^
 
+Susceptible to Exposed
+""""""""""""""""""""""
+
+Method One
+~~~~~~~~~~
+
+This method of modeling the susceptible to exposed state transition in the covid-19 model will be a replication of the transition probability used in the IHME forecast model such that the **incidence rate for an individual simulant at time-step t is defined as:**
+
+.. math:: \frac{\beta(t)S(I_1+I_2)^\alpha}{N}
+
+Where,
+
+.. list-table:: Parameter Definitions
+   :widths: 5 25
+   :header-rows: 1
+
+   * - Parameter
+     - Definition
+   * - :math:`S`
+     - Number of simulants in the :math:`S` state at time-step :math:`t`
+   * - :math:`I_1`
+     - Number of simulants in the :math:`I_1` state at time-step :math:`t`
+   * - :math:`I_2`
+     - Number of simulants in the :math:`I_2` state at time-step :math:`t`
+   * - :math:`N`
+     - Total number of simulants in the model at time-step :math:`t`
+   * - :math:`\beta(t)`
+     - Taken from IHME forecast model under stated assumptions regarding covariates
+   * - :math:`\alpha`
+     - Taken from IHME forecast model under stated assumptions
+
+.. note::
+
+	This method assumes random mixing of the population and assumes homogeneous transmission probability across all demographic groups.
+
+Method Two
+~~~~~~~~~~
+
+This method of modeling the susceptible to exposed state transition in the covid-19 model uses the :math:`\beta(t)` parameter *scaled to the average duration of the infectious period* OR :math:`R_t` from the IHME forecast model under stated assumptions regarding coviariates as the simulation's :math:`R_0` value under those same stated assumptions.
+
+.. warning::
+
+	This method requires careful scrutiny of the value that will be used for the simulation's :math:`R_0` to ensure that it is being applied in a consistent way
+
+At each time-step, for each simulant in the :math:`I_1` and :math:`I_2` covid-19 model states, sample the following number of simulants from the *entire simulant pool* (see *Important* note below for exception):
+
+.. math:: n_\text{simulants to sample} = R_0 * (\gamma_1 + \gamma_2) * duration_{\text{time step}}
```

Comment:
I don't understand this. Could you expand on it a little? Why does gamma_1 need to be there?

Reply:
We ended up abandoning this method, but I can still explain my thoughts here! I actually tried to simplify this equation and did it incorrectly. This is what it should be:

Both I1 and I2 are infectious states. 1/gamma_1 is the duration of the 1_2 state. 1/gamma_2 is the duration of the I_2 state.

R_0 is the value for the total number of people that a single infectious person will infect over the total amount of time they spend in the I_1 and I_2 states.

To scale this to a single time step (all duration measured in days)...

R_0 / (1/gamma_1 + 1/gamma_2) = number of infections per day during infectious period

R_0 / (1/gamma_1 + 1/gamma_2) * time-step duration = number of infectious per time-step during infectious period.